avr/cores/megacommand/A4/A4Sysex.cpp

```diff
@@ -15,7 +15,7 @@ void A4SysexListenerClass::start() {
 
 void A4SysexListenerClass::handleByte(uint8_t byte) {
 
-  if (isA4Message && MidiSysex2.len == sizeof(a4_sysex_hdr)) {
+  if (isA4Message && MidiSysex2.recordLen == sizeof(a4_sysex_hdr)) {
```

Comment:
these changes will also need to be made in the MD library

Reply:
the `handleByte` of listeners are never called?

Reply:
ignore. i thought len -> recordLen was variable name replacement.

---

handleByte wont be called on receive. we only process sysex messages when they are complete. by calling either end(); or end_immediate();

end_immediate() occurs during ISR.
end() occurs during main loop.